notification-publisher/src/test/java/org/dependencytrack/notification/NotificationRouterTest.java

```diff
@@ -264,7 +264,7 @@ void testResolveRulesWithDisabledRule() throws Exception {
 
         final var notification = Notification.newBuilder()
                 .setScope(SCOPE_PORTFOLIO)
-                .setGroup(GROUP_BOM_PROCESSED)
+                .setGroup(GROUP_NEW_VULNERABILITY)
```

Comment:
Yes, the test was supposed to verify that the `enabled` flag is honored by `NotificationRouter`.

But this notification was being tested against a rule that was configured for `GROUP_NEW_VULNERABILITY`:

https://github.com/DependencyTrack/hyades/blob/a2da593e146e8a5043b9408b61e2090521c985d8/notification-publisher/src/test/java/org/dependencytrack/notification/NotificationRouterTest.java#L260-L263

So the test succeeded, but for the wrong reason. `NotificationRouter#resolveRules` returned no rules because the group was mismatched, not because the rule was disabled...

Reply:
ohh okay :)